### PR TITLE
Update web pages

### PR DESCRIPTION
**Pull request by mr-tz**
extend text on home page and update to use `capa Explorer Web` throughout. 

preview
image

### Checklist

- [x] No CHANGELOG update needed

- [ ] No new tests needed

- [ ] No documentation update needed

### CODE REVIEW

**Review by williballenthin — APPROVED** [review]

File: web/explorer/src/components/DescriptionPanel.vue [hunk 1]
```diff
@@ -32,6 +50,12 @@
                 >.
             </p>
         </div>
+        <div>
+            <p class="text-base p-3 max-w-75rem" style="max-width: 75ch">
```
Comment (s-ff) [inline web/explorer/src/components/DescriptionPanel.vue:54]:
```suggestion
            <p class="text-base p-3 max-w-75rem">
```
remove this so sentence can take entire line?

File: web/explorer/src/components/DescriptionPanel.vue [hunk 2]
```diff
@@ -3,27 +3,45 @@
...
+                <li>Load the JSON results file into capa Explorer Web</li>
             </ol>
             <p class="text-700 text-sm m-0">
                 For more detailed information, explore the
                 <a
-                    href="https://github.com/mandiant/capa"
+                    href="https://github.com/mandiant/capa/tree/master/web/explorer"
                     target="_blank"
                     rel="noopener noreferrer"
                     class="text-blue-600 hover:text-blue-800 transition-colors duration-200"
```
Comment (s-ff) [inline web/explorer/src/components/DescriptionPanel.vue:47]:
```suggestion
                    class="underline"
```
for consistency with the two other links

Comment (s-ff) [reply web/explorer/src/components/DescriptionPanel.vue:47]: also remove this line

https://github.com/mandiant/capa/blob/1a467455ba7cd938cd323dc214a342f5a8497880/web/explorer/src/components/DescriptionPanel.vue#L66

<details><summary>so the underlines aren't hidden behind </summary>

grafik

</details>

**Review by s-ff — COMMENTED** [review]
Thanks!
A couple comments for your review

File: web/explorer/src/components/DescriptionPanel.vue [hunk 3]
```diff
@@ -16,13 +16,22 @@
...
+                    >, e.g.
+                    <ul>
+                        <li>
+                            download the latest
+                            <a href="https://github.com/mandiant/capa/releases" class="underline"
+                                >standalone executable release</a
+                            >
+                        </li>
+                        <li>or run <code>$ pip install flare-capa</code></li>
+                    </ul>
```
Comment (williballenthin) [inline web/explorer/src/components/DescriptionPanel.vue:31]: I don't like this reformatting, especially the hanging `>` characters.

Comment (s-ff) [reply web/explorer/src/components/DescriptionPanel.vue:31]: https://stackoverflow.com/questions/56291245/how-do-you-stop-prettier-in-vs-code-splitting-attributes-onto-multiple-lines recommends increasing the print width

Comment (mr-tz) [reply web/explorer/src/components/DescriptionPanel.vue:31]: @s-ff how do you recommend to run the linter and write the changes?

Comment (s-ff) [reply web/explorer/src/components/DescriptionPanel.vue:31]: After adding this option in `.prettier.json`, simply run `npm run fomat` to apply formatting. 


To inspect all available commands, run `npm run`.

File: rules
Comment (mr-tz) [inline rules:1]: - [ ] TODO: undo these

File: tests/data
Comment (mr-tz) [inline tests/data:1]: - [ ] TODO: undo these

File: web/explorer/src/components/DescriptionPanel.vue (hunk 3 above)
Comment (mr-tz) [reply web/explorer/src/components/DescriptionPanel.vue]: thanks, can you take another look now at this PR?

Comment (s-ff) [reply web/explorer/src/components/DescriptionPanel.vue]: Thanks, this seems to resolve the issue.

**Review by s-ff — APPROVED** [review]
Thank you very much.

**Pull request merged**